code/_compile_options.dm

```diff
@@ -1,8 +1,8 @@
 #define DEBUG
-//#define TESTING
```

Comment:
I dont think committing this part was intentional

Reply:
Just fixed that, thanks

Reply:
This is why you dont leave tabs open